package.json

```diff
@@ -24,4 +24,4 @@
 		"type": "git",
 		"url": "https://git.io/tcgen"
 	}
-}
+
```

Comment:
I request you to please not mess up with package.json unnecessarily.
You removed the last brace. 😕